pkg/database/token_test.go

```diff
@@ -131,13 +131,23 @@ func TestIssueToken(t *testing.T) {
 	acceptConfirmed := api.AcceptTypes{
 		api.TestTypeConfirmed: struct{}{},
 	}
+	acceptConfirmedAndSelfReport := api.AcceptTypes{
+		api.TestTypeConfirmed:  struct{}{},
+		api.TestTypeUserReport: struct{}{},
+	}
+
+	validNonce := generateNonce(t)
 
 	db, _ := testDatabaseInstance.NewDatabase(t, nil)
 
 	realm, err := db.FindRealm(1)
 	if err != nil {
 		t.Fatal(err)
 	}
+	realm.EnableUserReport()
```

Comment:
Sorry, even after a second pass, I dislike this API. For some reason, I think this is actually saving the realm instead of just modifying struct fields.

Reply:
renamed to `AddUserReportToAllowedTestTypes`

better?